ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ACSReader.scala

```diff
@@ -133,7 +170,30 @@ class ACSReader(
         )
       )
       .mapAsync(config.maxParallelPayloadCreateQueries)(fetchPayloads)
+      .mapAsync(config.contractProcessingParallelism)(
```

Comment:
Moved from [TransactionsReader.scala](https://github.com/digital-asset/daml/compare/pbatko/acs-reader-to-process-payloads?expand=1#diff-e2867a9d601aee55e86b4d83aba0b5dd5ca9034478f7e6ef5a1a6f4d2f3e1fea)